### ISSUE TITLE
Adding priority argument to speedbump.

### ISSUE THREAD

**Issue by noppanit** [fusioneng/speed-bumps#40]
It's hard to add speedbump in order. It would be a good idea to have priority of which speedbump should appear first. This could be like priority in WordPress. 

Thoughts? @goldenapples @danielbachhuber

**Comment by danielbachhuber** [comment]
:100:

**Comment by goldenapples** [comment]
Yes. That would be great. Currently they're just processed in order they're added, It would be good to sort by priority before processing.

**Comment by noppanit** [comment]
Just a quick question. I'm not sure how WordPress does this. but on top of my head I would have thought I would just sort the list of speedbumps by priority. Would that be enough?

**Comment by goldenapples** [comment]
> I would have thought I would just sort the list of speedbumps by priority

That sounds good to me. Core uses the priority [as an array key](https://github.com/WordPress/WordPress/blob/master/wp-includes/plugin.php#L85), and then just loops through the array, but that seems sort of clumsy to me. Just sorting by priority at the beginning of the insert_speed_bumps function should work fine, I would think.

**Comment by danielbachhuber** [comment]
> Core uses the priority as an array key, and then just loops through the array

Just to be clear, core also has `_wp_filter_build_unique_id()` to let two `priority=>10` items exist, which we should support as well.

Could it be as easy as #41?
